tests/test_orbit.py

```diff
@@ -1935,6 +1986,7 @@ def test_musun():
     # test_light_time_correction()
     test_find_passes()
     # test_musun()
+    # test_MG_2_3()
```

Comment:
Why is this commented out? Why create a test function that doesn't get run?

Reply:
I commented out the test because the other test calls were commented out, though I am not sure why the other tests are commented out.  I suggest I remove this line from the pull request until I understand the testing logic better.